README.md

```diff
@@ -15,7 +15,11 @@ In vanilla PHP, simply require the `Client` class and create a new instance with
 ```php
   use DoSomething\StatHat\Client as StatHat;
   
-  $stathat = new StatHat(['ez_key' => '...', 'user_key' => '...']);
+  $stathat = new StatHat([
+    'enabled' => (getenv('APP_ENV') == 'production'),
```

Comment:
Aaaaaaactually, I think this might make more sense as an optional `debug` parameter. :sweat_smile: 

My thought process is that adding an `enabled` flag would be a breaking change for existing installs, whereas if `debug => true` is not passed (since it's a new option in the next release) then behavior would stay consistent.